### PR TITLE
Add cube branding

### PR DESCRIPTION
**Pull request by thabarbados**
### Fix or Enhancement?
add cube branding and change readme header style

- [x] All tests passed

### Environment
- OS: BigSur MacOs
- NPM Version: 8.5.1

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,10 +1,14 @@
...
+<img align="right" width="150" height="150" alt="vue-chartjs logo" src="/assets/vue-chartjs.png">
+
+**vue-chartjs** is a wrapper for [Chart.js](https://github.com/chartjs/Chart.js) in vue. You can easily create reuseable chart components.
+
+Supports Chart.js v3 and v2.
 
 [![npm version](https://badge.fury.io/js/vue-chartjs.svg)](https://badge.fury.io/js/vue-chartjs)
 [![codecov](https://codecov.io/gh/apertureless/vue-chartjs/branch/master/graph/badge.svg)](https://codecov.io/gh/apertureless/vue-chartjs)
-[![Build Status](https://travis-ci.org/apertureless/vue-chartjs.svg?branch=master)](https://travis-ci.org/apertureless/vue-chartjs)
+[![Build Status](https://travis-ci.org/apertureless/vue-chartjs.svg?branch=master)](https://github.com/apertureless/vue-chartjs/actions)
```
Comment (dangreen) [inline README.md:11]:
```suggestion
[![Build Status](https://img.shields.io/github/workflow/status/apertureless/vue-chartjs/CI.svg)](https://github.com/apertureless/vue-chartjs/actions)
```

**Review by dangreen — CHANGES_REQUESTED** [review]

File: README.md [hunk 2]
```diff
@@ -14,15 +18,25 @@
...
+<span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+<a href="#demo">Demo</a>
+<span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+<a href="https://slack.cube.dev/?ref=eco-vue-chartjs">Slack</a>
+<span>&nbsp;&nbsp;•&nbsp;&nbsp;</span>
+<a href="https://stackoverflow.com/questions/tagged/vue-chartjs">Stack Overflow</a>
+<br />
+<hr />
+
+## Docs
```
Comment (dangreen) [inline README.md:38]: let's move it after "install" section?

File: README.md (hunk 1 above)
Comment (thabarbados) [reply README.md]: changed

File: README.md (hunk 2 above)
Comment (thabarbados) [reply README.md]: moved

**Review by dangreen — APPROVED** [review]

**Pull request merged**